java/src/main/java/com/codurance/training/tasks/actions/TaskActionHandlerImpl.java

```diff
@@ -0,0 +1,94 @@
+package com.codurance.training.tasks.actions;
+
+import com.codurance.training.tasks.Task;
+
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class TaskActionHandlerImpl implements TaskActionHandler {
+
+    private final PrintWriter out = new PrintWriter(System.out);
```

Comment:
this should not be created
Use the instance from main()

Reply:
Done